src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is doing its work.");

```

Comment:
The message should be more informative and indicate the type of `Machine` that is working. Consider specifying that a `Bulldozer` is operating.